foreign_cc/private/framework/toolchains/linux_commands.bzl

```diff
@@ -131,33 +125,30 @@ if [[ -z "$2" ]]; then
   exit 1
 fi
 local target="$2"
-mkdir -p "$target"
-if [[ -f "$1" ]]; then
-  # In order to be able to use `replace_in_files`, we ensure that we create copies of specfieid
-  # files so updating them is possible.
-  if [[ "$1" == *.pc || "$1" == *.la || "$1" == *-config || "$1" == *.mk || "$1" == *.cmake ]]; then
-    dest="$target/$(basename \"$1\")"
-    cp "$1" "$dest" && chmod +w "$dest" && touch -r "$1" "$dest"
-  else
-    ln -sf "$1" "$target/${1##*/}"
-  fi
-elif [[ -L "$1" && ! -d "$1" ]]; then
-  cp -pR "$1" "$2"
-elif [[ -d "$1" ]]; then
-  SAVEIFS=$IFS
-  IFS=$'\n'
-  local children=($(find -H "$1" -maxdepth 1 -mindepth 1))
-  IFS=$SAVEIFS
-  local dirname=$(basename "$1")
-  mkdir -p "$target/$dirname"
-  for child in "${children[@]:-}"; do
-    if [[ -n "$child" && "$dirname" != *.ext_build_deps ]]; then
-      ##symlink_to_dir## "$child" "$target/$dirname"
-    fi
-  done
-else
-  echo "Can not copy $1"
-fi
+# we symlink the ext_build_deps as well but we delete it after :)
+# this is a huge performance improvement than the original recursive version
+# so these extra copies are an okay performance loss
+cp -prsL "$1" --no-target-dir "$target"
+SAVEIFS=$IFS
+IFS=$'\n'
+local bad_directories=($(find -L "$target" -type d -name "*.ext_build_deps" -prune))
+IFS=$SAVEIFS
+for b in "${bad_directories[@]}"; do
+    rm -r "$b"
+done
```

Comment:
Changed it to `find` `exec` I hope this will fix the CI
I'm having a hard time debugging this because I need a working setup to run the failed tests (ndk etc)

Reply:
Yeah the ndk thing is annoying as with older bazel versions it wasn't a hard fail like it is now in 5.x. Most of us just push regularly against the CI to test changes that are hard to test locally due to lack of access to a specific system. All the changes get squashed on merge so intermediate commits don't matter.